tf/chunkparser.py

```diff
@@ -278,24 +388,36 @@ def convert_v5_to_tuple(self, content):
                  self.flat_planes[1]
```

Comment:
A few lines above here dep_ply_count is used - your v6 unpack line unpacks that into a variable called invariance_info you should rename as appropriate.

Reply:
should probably be rechecked - i think this is fixed now except the dummy/winner issue

Reply:
`        if (input_format == 132 or input_format == 133) and dep_ply_count >= 128:`